README.md

```diff
@@ -185,17 +185,42 @@ password_store_dir=/home/me/.work-passwords
 ### Password
 
 As usual with [pass], this helper assumes that the password is contained in the first line of the password store entry.
-Though uncommon, it is possible to strip a prefix from the data of the first line (such as `password:` by specifying an amount of characters to leave out in the `skip_password` field for an entry or also in the `[DEFAULT]` section to apply for all entries:
```

Comment:
Maybe the hint for specifying `skip_password` in the default section can be re-added somewhere.